features/storage/FEATURE_STORAGE/TESTS/cfstore/create/create.cpp

```diff
@@ -75,61 +75,61 @@ UVISOR_BOX_CONFIG(cfstore_create_box1, UVISOR_BOX_STACK_SIZE);
 #define CFSTORE_CREATE_TEST_01_TABLE_TAIL_ENTRY_04     { "There.is.in.the.worst.of.fortune.the.best.of.chances.for.a.happy.change", "Iphigenia.in.Tauris"}
 
 static cfstore_kv_data_t cfstore_create_test_01_data[] = {
-        CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_01,
-        CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_02,
-        CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_03,
-        { NULL, NULL},
+    CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_01,
+    CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_02,
+    CFSTORE_CREATE_TEST_01_TABLE_MID_ENTRY_03,
+    { NULL, NULL},
```

Comment:
Not sure what you require here. Astyle left this untouched. Changing to {NULL, NULL}. Hope this is as intended.